Dockerfile

```diff
@@ -17,14 +19,14 @@ ENV PATH="/virtualenv/bin:$PATH"
 RUN pip install \
     /srcdir \
     cx-Oracle \
-    ibm-db-sa \
     mysqlclient \
     psycopg2 \
     pymssql \
-    pyodbc
+    pyodbc \
+    $(arch | sed -e 's/x86_64/ibm-db-sa/g' -e 's/aarch64//g')
```

Comment:
I think this can be done similarly to the [snap setup](https://github.com/albertodonato/query-exporter/blob/main/snap/snapcraft.yaml#L60-L62) by changing the `ibm-db-sa` line above to:
```
ibm-db-sa; platform_machine == 'x86_64'
ibm-db-sa; platform_machine == 'ppc64le'
bm-db-sa; platform_machine == 's390x'
```